bc_obps/mocks/migrations/0003_set_mocked_time_in_transaction.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 5.1.14 on 2025-11-21 17:59
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mocks', '0002_mock_now_method'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+BEGIN;
+
+  create or replace function mocks.set_mocked_time_in_transaction(mocked_timestamp timestamptz)
+  returns void as
+  $function$
+    declare
+      parsed_epoch text;
+    begin
+
+      set search_path to mocks,public,pg_catalog;
+      -- Converting timestamptz to unix epoch
+      parsed_epoch := extract(epoch from mocked_timestamp)::text;
+      perform set_config('mocks.mocked_timestamp', parsed_epoch, false);
+
+    end;
+
+  $function$ language plpgsql volatile;
+
+COMMIT;
+""",
+            reverse_sql="drop function mocks.now();",
```

Comment:
Should this be `reverse_sql="drop function mocks.set_mocked_time_in_transaction(timestamptz);",`?